### PR TITLE
Feature/improve missing variants logging

### PR DESCRIPTION
**Pull request by redross**
@mmozuras what do you think?

### CODE REVIEW

File: lib/ab/assigned_test.rb [hunk 1]
```diff
@@ -46,6 +46,10 @@ def variants
       @test.variants.map(&:name)
     end
 
+    def name
```
Comment (mmozuras) [inline lib/ab/assigned_test.rb]: Why would this need to be part of public API?

Comment (redross) [inline lib/ab/assigned_test.rb]: Could be private.
Could also be removed, and something like `@test.try(:name)` be used in `MissingVariant`, though that seems like peeking into internal a bit too much?

Comment (mmozuras) [inline lib/ab/assigned_test.rb]: Private seems fine.

**Comment by mmozuras** [comment]
:+1:

**Pull request merged**